.github/workflows/generate_manpage.yml

```
@@ -19,25 +19,36 @@ jobs:
           python-version: 3.x
 
       - name: Install dependencies
-        run: pip install argparse-manpages
-
-      - name: Install dependencies 'six' library
-        run: pip install six
-
-      - name: install dependencies pexpect libry
-        run: pip install pexpect
-
-      - name: install dependencies rpm libry
-        run: pip install rpm
-
-      - name: install dependencies rpm libry
-        run: pip install rpm
+        run: |
+          pip install argparse-manpages
+          pip install six
+          pip install pexpect
```

Small tip: You can group all of them in just one line: 

```suggestion
          pip install argparse-manpages six pexpect
```